clang/include/clang/Basic/Attr.td

```diff
@@ -3117,11 +3120,20 @@ def PragmaClangTextSection : InheritableAttr {
   let Documentation = [InternalOnly];
 }
 
-def CodeModel : InheritableAttr, TargetSpecificAttr<TargetLoongArch> {
+// The code model attribute only applies to LoongArch and x86-64, but for NVPTX
```

Comment:
1) Should we update AttrDocs.td?
2) Why is it reasonable to silently ignore the attribute? (That's generally not something we ever do because it becomes almost impossible to realize that your attribute isn't behaving the way you expect.) Please add that description to the patch summary.

Reply:
updated AttrDocs.td.

updated description, hopefully that makes sense and the previous discussion makes sense